src/test/resources/org/jpeek/metricstest-params.csv

```diff
@@ -36,10 +36,7 @@ TwoCommonAttributes,NHD,0.3333d
 OneVoidMethodWithoutParams,NHD,NaN
 WithoutAttributes,NHD,0.0d
 OneMethodCreatesLambda,NHD,0.0d
-MethodsWithDiffParamTypes,CCM,0.0667d
```

Comment:
@HDouss You could comment out this and below deleted lines.
The `@CsvFileSource` supports `#` for commenting